### ISSUE TITLE
[question] 

### ISSUE THREAD

**Issue by konstantinsp** [mui/mui-x#13955]
### Order ID or Support key 💳

86076

### Search keywords

aggregation on top

### Latest version

- [X] I have tested the latest version

### The problem in depth

Hello,
is there a way to position aggregation row on top (as first row) instead of bottom?

### Your environment

<details>
  <summary>`npx @mui/envinfo`</summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Comment by michelengelen** [comment]
@KenanYusuf could you have a look please?

**Comment by cherniavskii** [comment]
Hi @konstantinsp 
This is not supported at the moment.
This is the first time we got this request, so I'm adding a "waiting for upvotes" label to see if there's a demand for this.

**Comment by karopolopoulos** [comment]
Yeah this is something we are interested in as well. Im considering using `gridAggregationLookupSelector` to pin my own row at the top for now.

I noticed an old closed thread asking this question: https://github.com/mui/mui-x/issues/7129#issuecomment-1342917200
